python/lsst/analysis/tools/analysisParts/baseSources.py

```diff
@@ -0,0 +1,33 @@
+from lsst.analysis.tools.actions.scalar import UniqueAction
+from lsst.analysis.tools.actions.vector import DownselectVector, LoadVector, ThresholdSelector, VectorSelector
+from lsst.analysis.tools.interfaces import AnalysisTool
+
+
+class BaseSources(AnalysisTool):
+    """Base class for counting associated and unassociated sources."""
+
+    def setDefaults(self):
+        super().setDefaults()
+
+        self.process.buildActions.loadVectorSources = LoadVector()
+        self.process.buildActions.associatedVectorSelector = ThresholdSelector()
+        self.process.buildActions.unassociatedVectorSelector = ThresholdSelector()
+
+        # assign keys for PSF and AP Flux
```

Comment:
What do the next 7 lines have to do with PSF and AP flux?

Reply:
This was a stray comment from a notebook. I'll remove it.